ui/src/app/components/runtime-configuration-panel/disk-selector.tsx

```diff
@@ -37,6 +67,26 @@ export const DiskSelector = ({
     <FlexColumn
       style={{ ...styles.controlSection, gap: '11px', marginTop: '11px' }}
     >
+      {!disableDetachableReason && (
+        <FlexRow style={warningBoxStyle.box}>
+          <WarningIcon style={warningBoxStyle.icon} />
+          <div>
+            <AoU /> will now only support re-attachable persistent disks as the
```

Comment:
let's work on the text here.

How about "requires the use of reattachable persistent disks when using Standard VMs"

Specifically I want it to sound like it's describing a fact about the state of the world, because that's less scary than a **change** - but we should discuss that with comms